setup.py

```diff
@@ -15,6 +15,6 @@
         'Programming Language :: Python :: 3.7'
     ],
     packages=find_packages(exclude=['docs', 'tests']),
-    install_requires=['tensorflow==1.13', 'matplotlib', 'numpy', 'comet-ml', 'scikit-learn'],
-    extras_require={'test': ['flake8', 'pytest', 'pytest-cov', 'pillow']}
+    install_requires=['tensorflow==1.13', 'matplotlib', 'numpy', 'comet-ml', 'scikit-learn', 'pillow'],
```

Comment:
Why was pillow moved to install_requires? It isn't used in the main code, only in the test suite.